source/_lovelace/picture-elements.markdown

```diff
@@ -214,26 +249,54 @@ entity:
   type: string
 tap_action:
   required: false
-  description: more-info, toggle, navigate, call-service
-  type: string
-  default: more-info
+  description: Action to take on tap
+  type: object
+  keys:
+    action:
+      required: true
+      description: Action to perform (`more-info`, `toggle`, `call-service`, `navigate`, `none`)
+      type: string
+      default: `none`
```

Comment:
```suggestion
      default: "`none`"
```

Reply:
@klaasnicolaas why do you think they should be quotes surrounding it?